packages/sanity/src/structure/panes/documentList/DocumentListPane.tsx

```diff
@@ -113,7 +113,10 @@ export const DocumentListPane = memo(function DocumentListPane(props: DocumentLi
   } = useDocumentList({
     apiVersion,
     filter,
-    perspective: perspectiveStack.length === 0 ? 'raw' : perspectiveStack,
+    perspective:
+      selectedPerspective === 'drafts' || selectedPerspective === 'published'
```

Comment:
It would be very helpful if `usePerspective()` returned the result of this expression, so that we don't have to replicate it everywhere we need to get something using the current perspective

Reply:
Thanks for the review and the suggestion @mariusGundersen, we'll do this in a follow-up PR  🙇🏼